.github/workflows/build-publish-pypi.yml

```diff
@@ -1,12 +1,9 @@
-# This workflow will build and publish the package to PyPI upon push or merge to main branch
-# and then create a GitHub release based on latest release tag
-
 name: Build and Release
 
 on:
   push:
-    tags:
-      - release/*
+    branches:
+      - 'main'
```

Comment:
no don't do this. follow the same thing we have in sdk. add the `create-release-tag` workflow.